cdk.json

```diff
@@ -66,26 +66,26 @@
     "dev": {
         "VPC_CIDR": "10.254.192.0/24",
         "FQDN": "dev.schematic.io",
-        "CERTIFICATE_ARN": "arn:aws:acm:us-east-1:631692904429:certificate/0e9682f6-3ffa-46fb-9671-b6349f5164d6"
-      },
+        "CERTIFICATE_ARN": "arn:aws:acm:us-east-1:631692904429:certificate/0e9682f6-3ffa-46fb-9671-b6349f5164d6",
```

Comment:
You might add the container images and ports to the configuration.

Reply:
Maybe having VARs in this configuration (cdk.json) file and an in the code (app.py) is a bit confusing.  Would consolidating all VARs to the CDK code (in app.py or some common python object) make the setup less confusing?

Reply:
I just noticed that `cdk.json` in this repo' contains references to OpenChallenges resources. (See the `secrets` part of the file.). Can you please remove them?

Reply:
> Maybe having VARs in this configuration (cdk.json) file and an in the code (app.py) is a bit confusing. Would consolidating all VARs to the CDK code (in app.py or some common python object) make the setup less confusing?

Yes, that's not a bad idea: The application parameters could go in `app.py`, where they are used.

Reply:
done